src/query/storages/parquet/src/parquet2/statistics.rs

```diff
@@ -104,7 +104,7 @@ impl BatchStatistics {
             unsafe { self.max_values.index_unchecked(index).to_owned() },
             self.null_count[index],
             0, // this field is not used.
-            self.distinct_count.as_ref().map(|d| d[index]),
+            None,
         )
```

Comment:
Do we need to call `with_distinct_of_values` with `self.distinct_count` here ?

Reply:
Yes, but it's a little trivial.

1. pruning doesn't need the `distinct_of_value` field.
2. parquet2 is being deprecated soon.

Reply:
Got it.